scripts/main/dragDrop.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * drag album to another one
+ * @param {DragEvent} ev
+ */
+function startAlbumDrag(ev) {
+	ev.dataTransfer.setData("text", ev.target.id);
```

Comment:
You should be able to extract what you need without defining `id`, using `dataset.id` and `className`, I guess?